resources/Doxyfile.in

```diff
@@ -0,0 +1,344 @@
+# Doxyfile 1.8.15
```

Comment:
As a "generic" utility, this configuration is pretty opinionated on how it lays out Doxygen.  I guess we can always add configurability later, so I'm OK with this for now.

Reply:
That's what `CONFIG_OVERLAY` is meant for. The idea was to provide reasonable defaults to reduce duplication, but downstream packages can always override all of them. Including the ones that make this work, so it should probably be documented what must be preserved.